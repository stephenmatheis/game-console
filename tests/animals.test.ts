test('snapshot test with CMK', () => {
    expect(true).toMatchSnapshot();
});
