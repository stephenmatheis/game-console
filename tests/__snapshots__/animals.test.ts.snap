// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test with CMK 1`] = `true`;
